test/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.Tests/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.Tests.csproj

```diff
@@ -18,4 +18,8 @@
     <PackageReference Include="xunit.runner.visualstudio" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
Don't think you need this

Reply:
VS adds these automatically

@Eilon are we ignoring these?

Reply:
If you remove it and VS keeps putting it back then keep it.

Reply:
Opening the solution in VS re-adds these. So checking them in